nailgun/entities.py

```diff
@@ -844,9 +844,11 @@ def __init__(self, server_config=None, **kwargs):
             'command': entity_fields.StringField(
                 required=True,
                 str_type='latin1',
+                default='top',
```

Comment:
I vote for not adding this because there are some images that does not provide top like default centos.